tests/suites/test_suite_ssl.data

```diff
@@ -2949,26 +2949,26 @@ ssl_decrypt_non_etm_cbc:MBEDTLS_CIPHER_CAMELLIA_128_CBC:MBEDTLS_MD_SHA384:0:255
 SSL TLS 1.3 Key schedule: Secret evolution #1
 # Vector from TLS 1.3 Byte by Byte (https://tls13.ulfheim.net/)
 # Initial secret to Early Secret
-depends_on:PSA_WANT_ALG_SHA_256
+depends_on:PSA_WANT_ALG_SHA_256:MBEDTLS_ECP_DP_CURVE25519_ENABLED
```

Comment:
Indeed, since it's TLS 1.3 code, it should use PSA constants.

Note that building TLS, even 1.3, without `ECP_C` is not supported yet, that's actually the goal of an [upcoming EPIC](https://github.com/orgs/Mbed-TLS/projects/1#column-18986128). But that EPIC will only be easier if we try to use the correct constants (that is, for code specific to 1.3, PSA constants) in the meantime.

Reply:
The problem here is that for a case where `input` is NULL and `input_len` is zero, the input length is equal to hash length.
So, for a case where `PSA_RAW_KEY_AGREEMENT_OUTPUT_MAX_SIZE` is 24 bytes (SECP192 only), should `tmp_input` be `PSA_HASH_MAX_SIZE`?

Reply:
Yes, `MAX(PSA_HASH_MAX_SIZE, PSA_RAW_KEY_AGREEMENT_OUTPUT_MAX_SIZE)` looks correct to me.